### PR TITLE
feature: KIDS-1216 Direct child to recommendation flow

### PR DESCRIPTION
**Pull request by Daniela510**

### CODE REVIEW

File: lib/core/network/api_service.dart [hunk 1]
```diff
@@ -139,4 +139,22 @@ class APIService {
       return jsonDecode(response.body) as Map<String, dynamic>;
     }
   }
+
+  Future<bool> sendRecEmail({required String id}) async {
```
Comment (pavelpavelko) [inline lib/core/network/api_service.dart:143]: just a tiny style suggestion - avoid abbreviation in names, especially when a name abbreviation might have multiple meanings, like Rec, is it something about recording? :)

File: lib/features/recommendation/cubit/recommendation_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      eventName: AmplitudeEvent.askToFindCharityPressed,
+    );
+    try {
+      final response = await _recRepositoy.sendRecEmail(id: kidId);
+      if (response) {
+        emit(RecommendationSent());
+      } else {
+        emit(RecommendationSendFailed());
+      }
+      emit(RecommendationSent());
```
Comment (pavelpavelko) [inline lib/features/recommendation/cubit/recommendation_cubit.dart:24]: a bit unclear logic here. 
why do we need to emit RecommendationSent despite of the response value?

File: lib/features/recommendation/widgets/ask_rec_button.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/features/recommendation/cubit/recommendation_cubit.dart';
+
+class AskMyParentsButton extends StatelessWidget {
+  const AskMyParentsButton({required this.completed, super.key});
+  final bool completed;
+  @override
+  Widget build(BuildContext context) {
+    final String kidId = context.read<ProfilesCubit>().state.activeProfile.id;
```
Comment (pavelpavelko) [inline lib/features/recommendation/widgets/ask_rec_button.dart:11]: super tiny comment also regarding naming - shall we keep consistency and name such vars either child (children) or kid (kids)?

**Review by pavelpavelko — APPROVED** [review]
lgtm with few small, mostly naming related suggestions

File: lib/features/recommendation/cubit/recommendation_cubit.dart (hunk 2 above)
Comment (Daniela510) [reply lib/features/recommendation/cubit/recommendation_cubit.dart:24]: It's a mistake. Thank you

**Pull request merged**